modules/KIWILinuxRC.sh

```diff
@@ -802,11 +803,17 @@ function udevStart {
             enablePlymouth=0
             break
             ;;
+        case "$o" in
+            fips=1*)
+            fipsEnabled=1
+            ;;
         esac
     done
     if [ $enablePlymouth -eq 1 ]; then
         startPlymouth
     fi
+    # start haveged
+    startHaveged
```

Comment:
iirc the plan is to start haveged if fips is enabled, correct ? if so I think the following is more clear

```bash
if [ $fipsEnabled -eq 1 ];then
    startHaveged
fi
```

or am I misunderstanding ?